website/server/controllers/api-v3/challenges.js

```diff
@@ -315,6 +323,14 @@ api.joinChallenge = {
     let chalLeader = await User.findById(response.leader).select(nameFields).exec();
     response.leader = chalLeader ? chalLeader.toJSON({minimize: true}) : null;
 
+    res.analytics.track('join challenge', {
+      uuid: user._id,
+      hitType: 'event',
+      category: 'behavior',
+      groupName: group.privacy === 'private' ? null : group.name,
+      groupType: group._id === TAVERN_ID ? 'tavern' : group.type,
```

Comment:
Challenge ID?

Also as a suggestion, if the analytics titles (? keys?) were "challenge create", "challenge John", etc, they'd all sort together in the Amplitude interface which would be convenient when doing several challenge investigations.

Reply:
Similarly below for using "task create" rather than "create task" in case we ever add "task delete" too (or "task copy" if we build that requested feature}.

It might be too late to use "task score" since "score task" is already in use.

Reply:
Good points! Given we're rebooting "score task" and haven't had any data in that event for some time, I think it'll be OK to have a rename.

I'll add another commit with these changes, maybe also add "challenge delete" and "challenge award" (challenge conclude?)

Reply:
Thanks!! :)

The usual term for ending a challenge with a winner is "close" (as opposed to "delete" which is ending with no winner).

Reply:
ah, I just missed that. I'll make it "close"